CHANGELOG.md

```diff
@@ -2,6 +2,40 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.2.0-beta.1 November 18, 2021
+
+## Features ✨ and improvements 🏁
+* Remove `MapView` argument when constructing `AnnotationManager`. ([#766](https://github.com/mapbox/mapbox-maps-android/pull/766))
+* Implement cluster API on top of `MapboxMap.queryFeatureExtensions`. ([#773](https://github.com/mapbox/mapbox-maps-android/pull/773))
+* Introduce view annotation support which allows adding Android views on top of the `MapView` anchored to geo-point. ([#834](https://github.com/mapbox/mapbox-maps-android/pull/834))
+* Add heatmap and circle layer support to globe view. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
+* Improve performance for querying large number of features. ([#852](https://github.com/mapbox/mapbox-maps-android/pull/852))
```

Comment:
Could use some additional wording on how we were able to do that

Reply:
I have no idea how we were able to do that, it's about actually speeding up QRF function dramatically.